simple/stats/db.py

```diff
@@ -190,9 +192,13 @@ def commit_and_close(self):
         OBSERVATIONS_TMCF)
 
     # Not supported for main DC at this time.
```

Comment:
what does "not supported for main DC" mean?

Reply:
This will be called for generating the custom dc SVG cache. We don't build the SVG cache for main DC here so not supporting it.